qiskit/transpiler/preset_passmanagers/level0.py

```diff
@@ -145,8 +145,10 @@ def _swap_condition(property_set):
     elif routing_method == "none":
         _swap += [
             Error(
-                msg="No routing method selected, but circuit is not routed to device. "
-                "CheckMap Error: {check_map_msg}",
+                msg=(
+                    "No routing method selected, but circuit is not routed to device. "
+                    "CheckMap Error: {check_map_msg}"
```

Comment:
```suggestion
                    f"CheckMap Error: {check_map_msg}"
```

Reply:
Seems like there's a few places where f prefix was initially missing. I'll try to follow up to this PR with a mechanical check for that situation.

Reply:
This was actually correct and `Error` pass does it's own substitution